### PR TITLE
#1565 fix: additional links

### PR DESCRIPTION
**Pull request by legik**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [x] Your contribution is written in the 2nd person (e.g. you)
- [x] Your contribution is written in an active present form for as much as possible.
- [x] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [x] Your contribution has proper formatted markdown and/or code
- [x] Any references to website have been formatted as [TEXT](URL “NAME”)
- [x] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #1565.

Fix additional links, related to gitbook markdown parsing.

### CODE REVIEW

File: Document/0x06h-Testing-Platform-Interaction.md [hunk 1]
```diff
@@ -2803,7 +2803,7 @@ JavaScript code in a `WKWebView` can still send messages back to the native app
 
 Verify if a JavaScript to native bridge exists by searching for `WKScriptMessageHandler` and check all exposed methods. Then verify how the methods are called.
 
-The following example from ["Where's My Browser?"](https://github.com/authenticationfailure/WheresMyBrowser.iOS/blob/b8d4abda4000aa509c7a5de79e5c90360d1d0849/WheresMyBrowser/WKWebViewPreferencesManager.swift#L98 "Where's My Browser? WKWebViewPreferencesManager.swift Line 98") demonstrates this.
+The following example from ["W0x06h-testing-platform-interactionhere's My Browser?"](https://github.com/authenticationfailure/WheresMyBrowser.iOS/blob/b8d4abda4000aa509c7a5de79e5c90360d1d0849/WheresMyBrowser/WKWebViewPreferencesManager.swift#L98 "Where\'s My Browser? WKWebViewPreferencesManager.swift Line 98") demonstrates this.
```
Comment (sushi2k) [inline Document/0x06h-Testing-Platform-Interaction.md]:
```suggestion
The following example from ["Where's My Browser?"](https://github.com/authenticationfailure/WheresMyBrowser.iOS/blob/b8d4abda4000aa509c7a5de79e5c90360d1d0849/WheresMyBrowser/WKWebViewPreferencesManager.swift#L98 "Where\'s My Browser? WKWebViewPreferencesManager.swift Line 98") demonstrates this.
```

**Comment by legik** [comment]
I don't have any script yet, just reading the book and mark invalid links)
What should I do with you comment? Press resolve button?

**Comment by sushi2k** [comment]
You can just "commit suggestion", if you think it makes sense. Then there is no need for you to make the change locally and commit to the PR.

**Comment by sushi2k** [comment]
Thanks for your help @legik! That's really good work.

**Pull request merged**